kolibri/core/assets/src/views/AppBar.vue

```diff
@@ -104,6 +104,12 @@
               <LogoutSideNavEntry v-if="isUserLoggedIn" />
             </template>
 
+            <template #footer>
+              <!-- Only show this when on a SoUD -->
+              <div v-if="isSubsetOfUsersDevice" class="role" data-test="learnOnlyNotice">
```

Comment:
Oh, just noticed one other thing here - shouldn't we only show this to non-learners? Otherwise it's a slightly confusing notice "this is a learn only device, so you can only see the things you'd normally see!".